### PR TITLE
Install mui framework

### PR DESCRIPTION
**Pull request by OliaHrytsak**
Install MUI framework, added theme and Theme Provider

Added PageWrapper conponent and test for it

Added script for test coverage:

"test:coverage": "npm run test -- --collectCoverage"

### CODE REVIEW

File: code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import { render, screen } from "@testing-library/react";
+
+describe("PageWrapper", () => {
+  test("should render children", () => {
+    render(
+      <PageWrapper>
+        <div>Test children</div>
```
Comment (mhevyk) [inline code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx]: Maybe we can get rid of `<div>` to improve performance

File: code/tsconfig.json [hunk 2]
```diff
@@ -53,7 +53,7 @@
     "src/**/*",
     "config/**/*",
     "types/**/*"
-  ],
+, "src/styles/muiTheme.ts"  ],
```
Comment (mhevyk) [inline code/tsconfig.json]: Why is it used here?

File: code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      </PageWrapper>
+    );
+
+    expect(screen.getByText("Test children")).toBeInTheDocument();
+  });
```
Comment (dmtrth25) [inline code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx]: can create variable with `getByText` option and use it like
`expect(variable).toBeInTheDocument()` for more readability and we'll try to follow this convention for tests

File: code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx (hunk 1 above)
Comment (OliaHrytsak) [reply code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx]: Fixed!

File: code/tsconfig.json (hunk 2 above)
Comment (OliaHrytsak) [reply code/tsconfig.json]: it was added automatically( Removed

File: code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx (hunk 3 above)
Comment (OliaHrytsak) [reply code/src/layouts/app-wrapper/tests/pageWrapper.test.tsx]: Fixed. Thank you

**Review by D1nviz — APPROVED** [review]

**Review by heshvab — APPROVED** [review]

**Review by mhevyk — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Pull request merged**